stripe/api_resources/file_upload.py

```diff
@@ -7,7 +8,7 @@ class FileUpload(ListableAPIResource):
 
     @classmethod
     def api_base(cls):
-        return upload_api_base
+        return stripe.upload_api_base
```

Comment:
Opportunistic fix to ensure that the current value of `upload_api_base` is used (before, the value was only read once when the library is loaded).

Reply:
Nice find.